CPR/ServiceplatformenCprServiceResult.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/*
+ * This file is part of aakb/kontrolgruppen-core-bundle.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace Kontrolgruppen\CoreBundle\CPR;
+
+/**
+ * Class ServiceplatformenCprServiceResult.
+ */
+class ServiceplatformenCprServiceResult implements CprServiceResultInterface
+{
+    private $response;
+
+    /**
+     * ServiceplatformenCprServiceResult constructor.
+     *
+     * @param $response
+     */
+    public function __construct($response)
+    {
+        $this->response = $response;
+    }
+
+    /**
+     * Get first name.
+     *
+     * @return string
+     */
+    public function getFirstName(): string
+    {
+        return $this->response
+            ->persondata
+            ->navn
+            ->fornavn;
```

Comment:
This makes a lot of assumptions of the structure/value of `$this->response`. Maybe we need some additional checks, e.g. using https://symfony.com/doc/current/components/property_access.html.